packages/discord.js/src/structures/MessageReaction.js

```diff
@@ -43,13 +49,39 @@ class MessageReaction {
   }
 
   _patch(data) {
+    if ('burst_colors' in data) {
+      /**
+       * Hexadecimal colors used for this super reaction
+       * @type {string[]}
```

Comment:
It's possibly undefined otherwise, lets either default this to `[]` or mark it correctly as nullable (and handle that edgecase)

Reply:
It shouldnt be? [ddevdocs reaction object](https://discord.com/developers/docs/resources/message#reaction-object)

Reply:
That’s only for messageReactionAdd event. Not the case for messageReactionRemove or others where a (partial) MessageReaction can be created and/or patched from.